### ISSUE TITLE
Build also depend on zlib

### ISSUE THREAD

**Issue by michelepagot** [vysheng/tg#1477]
zlib missing detected by ./configure
Mind to cite it in readme

**Comment by Estagiarius** [comment]
In that case, open your Synaptic and install zlib (if not installed), and to work, zlibdev.
In my computer, this packages are called zlib1g (zlib) and zlib1g-dev (zlibdev).

**Comment by holms** [comment]
@Estagiarius this should be specified in readme.

**Comment by Estagiarius** [comment]
@holms I agree with you, but the project seems to be not maintained now. The last update seems to be two year ago.

**Comment by holms** [comment]
@Estagiarius oh crap :) wrote to @vysheng in telegram let's see what will happen

**Comment by Estagiarius** [comment]
I just hope that he updates the readme with all another issues too

**Comment by ya3er** [comment]
use this command to handle it:
apt-cyg install zlib-devel
